File: logica-programacao/python/014.py
"""Dado um número inteiro m (minutos):
Calcule:
Horas completas
Minutos restantes

Exemplo:
m = 187 → 3 horas e 7 minutos"""

numero = int(input("Digite um número em minutos: "))
horas_completas = numero // 60;
minutos_restantes = numero % 60

print(f"{numero} -> {horas_completas} horas e {minutos_restantes} minutos.")

File: logica-programacao/javascript/014.js
/*
Dado um número inteiro m (minutos):
Calcule:
Horas completas
Minutos restantes

Exemplo:
m = 187 → 3 horas e 7 minutos */

let numero = Number(prompt("Digite um número em minutos: "));
let horasCompletas = Math.floor(numero / 60);
let minutosRestantes = numero % 60;

console.log(`${numero} -> ${horasCompletas} horas e ${minutosRestantes} minutos.`);